compiler/il/OMRILOpCodeProperties.hpp

```diff
@@ -9177,6 +9177,22 @@
    /* .ifCompareOpCode      = */ TR::BadILOp,
    },
 
+  {
+   /* .opcode               = */ TR::newvalue,
+   /* .name                 = */ "newvalue",
+   /* .properties1          = */ ILProp1::HasSymbolRef,
+   /* .properties2          = */ ILProp2::CanRaiseException| ILProp2::MayUseSystemStack | ILProp2::New,
```

Comment:
This may not be an issue currently because there is no code that creates a newvalue node yet, but one would need to make sure that all the pieces of code that currently check isNew() would work fine with a newvalue node. Since @Charlmzz is likely to do the work to create this type of node in the future, I thought I'd mention it here.